src/main/java/com/github/markusbernhardt/seleniumlibrary/keywords/Element.java

```diff
@@ -83,15 +84,23 @@ public void currentFrameShouldNotContain(String text, String...params) {
 	@RobotKeyword("Verify the element identified by ``locator`` contains ``text``.\n\r"
 	        + "\n\r"
 	        + "See `Introduction` for details about locators.")
-	@ArgumentNames({ "locator", "text", "message=NONE" })
+	@ArgumentNames({ "locator", "text", "message=NONE", "ignore_case=False" })
 	public void elementShouldContain(String locator, String text, String...params) {
-	    String message = robot.getParamsValue(params, 0, "");
+        String message = robot.getParamsValue(params, 0, "");
+        Boolean ignoreCase = robot.getParamsValue(params, 1, Boolean.FALSE);
 		String actual = getText(locator);
+		String textToCompare = text;
+		if (ignoreCase) {
+			actual = actual.toLowerCase();
+			textToCompare = textToCompare.toLowerCase();
```

Comment:
Using toLowerCase()/toUpperCase() without specifying a locale leads to bugs like that: https://garygregory.wordpress.com/2015/11/03/java-lowercase-conversion-turkey/

:-/